Source/Scene/ClassificationModel.js

```diff
@@ -0,0 +1,1869 @@
+        this._readyPromise = when.defer();
+
+        /**
+         * This property is for debugging only; it is not for production use nor is it optimized.
+         * <p>
+         * Draws the bounding sphere for each draw command in the model.  A glTF primitive corresponds
+         * to one draw command.  A glTF mesh has an array of primitives, often of length one.
+         * </p>
+         *
+         * @type {Boolean}
+         *
+         * @default false
+         */
+        this.debugShowBoundingVolume = defaultValue(options.debugShowBoundingVolume, false);
+        this._debugShowBoundingVolume = false;
+
+        /**
+         * This property is for debugging only; it is not for production use nor is it optimized.
+         * <p>
+         * Draws the model in wireframe.
+         * </p>
+         *
+         * @type {Boolean}
+         *
+         * @default false
+         */
+        this.debugWireframe = defaultValue(options.debugWireframe, false);
+        this._debugWireframe = false;
+
+        this._distanceDisplayCondition = options.distanceDisplayCondition;
+        this._classificationType = options.classificationType;
+
+        // Undocumented options
+        this._vertexShaderLoaded = options.vertexShaderLoaded;
+        this._classificationShaderLoaded = options.classificationShaderLoaded;
+        this._uniformMapLoaded = options.uniformMapLoaded;
+        this._pickVertexShaderLoaded = options.pickVertexShaderLoaded;
+        this._pickFragmentShaderLoaded = options.pickFragmentShaderLoaded;
+        this._pickUniformMapLoaded = options.pickUniformMapLoaded;
+        this._ignoreCommands = defaultValue(options.ignoreCommands, false);
+        this._upAxis = defaultValue(options.upAxis, Axis.Y);
+        this._batchTable = options.batchTable;
+
+        /**
+         * @private
+         * @readonly
+         */
+        this.cull = defaultValue(options.cull, true);
+
+        this._computedModelMatrix = new Matrix4(); // Derived from modelMatrix and axis
+        this._initialRadius = undefined;           // Radius without model's scale property, model-matrix scale, animations, or skins
+        this._boundingSphere = undefined;
+        this._scaledBoundingSphere = new BoundingSphere();
+        this._state = ModelState.NEEDS_LOAD;
+        this._loadResources = undefined;
+
+        this._mode = undefined;
+
+        this._perNodeShowDirty = false;            // true when the Cesium API was used to change a node's show property
+        this._dirty = false;                       // true when the model was transformed this frame
+        this._maxDirtyNumber = 0;                  // Used in place of a dirty boolean flag to avoid an extra graph traversal
+
+        this._runtimeNode = {
+            matrix : undefined,
+            translation : undefined,
+            rotation : undefined,
+            scale : undefined,
+
+            // Per-node show inherited from parent
+            computedShow : true,
```

Comment:
Here and below, does this still need to be simplified give that we will only support one mesh?  Did we confirm that only one primitive is OK as well?